src/app/core/core.module.ts

```diff
@@ -1,29 +1,41 @@
 import { HTTP_INTERCEPTORS } from "@angular/common/http";
 import { NgModule, Optional, Provider, SkipSelf } from "@angular/core";
+import { EffectsModule } from "@ngrx/effects";
+import { StoreModule } from "@ngrx/store";
+import { StoreDevtoolsModule } from "@ngrx/store-devtools";
+
 import { AuthGuard } from "./guards/index";
 import { AuthInterceptor, LoaderInterceptor } from "./interceptors/index";
+import { StoreFacadeService } from "./store-facade/store-facade.service";
+import { ServiceModule } from "./services/index";
 
+import * as fromApp from "./store/app.reducer";
 
-import { ServiceModule } from "./services/index";
+import * as fromEffects from "./store/effects/index";
```

Comment:
Also, you can create effects.ts file in core/effects/effects.ts, import all the effects and export an array of effects

Reply:
fixed, although i had to disable typedef  in effects/index.ts